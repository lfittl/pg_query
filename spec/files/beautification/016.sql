SELECT distinct a, b FROM c ORDER BY a
