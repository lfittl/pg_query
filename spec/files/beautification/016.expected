SELECT
  DISTINCT
  "a",
  "b"
FROM
  "c"
ORDER BY
  "a"
